docs/developers/v2/hacking-with-yearn.md

```diff
@@ -2,145 +2,153 @@
 
-Here's an example of integrating V2 vaults and ERC-4626. 
+Here's an example of integrating V2 vaults and ERC-4626.
 
 https://github.com/storming0x/ystarter-foundry-kit
 
 What you'll find:
+
 - Basic example Solidity Smart Contracts for integrating with Yearn Vaults for both V2 and ERC4626 interfaces.
 - ERC4626 adapter to wrap yearn vaults.
 - Example Contracts
 - Sample test suite.
 
-**Yearn Vaults**
+#### Yearn Vaults
 
 [This repository](https://github.com/yearn/yearn-vaults#readme) includes the set of smart contracts that are used for the Yearn Vaults. It contains the requirements, code, deployment scripts, and tests necessary for the core protocol, including an inheritable template strategy for use with Solidity-based strategies that interact with Yearn Vaults. These contracts are used to create a simple way to generate high risk-adjusted returns for depositors of various assets via best-in-class lending protocols, liquidity pools, and community-made yield farming strategies on Ethereum.
 
 ## Testing
 
 ### Introduction
-You can test Yearn vaults without spending cryptoassets on the available testnet for each network. 
+
+You can test Yearn vaults without spending cryptoassets on the available testnet for each network.
 
 To get started, head to the yearn testnet client which is available at: https://hack.yearn.fi/
 
-Make sure to switch your wallet provider to the appropriate testnet network in order to use the protocol testnet without incurring costs on the mainnet network. 
- 
-Testnet networks are different environments from mainnet and simulate real use cases for development, testing or production. The assets on a testnet are not “real,” meaning that they cannot be withdrawn to your wallet for any real economic value. 
- 
+Make sure to switch your wallet provider to the appropriate testnet network in order to use the protocol testnet without incurring costs on the mainnet network.
+
+Testnet networks are different environments from mainnet and simulate real use cases for development, testing or production. The assets on a testnet are not “real,” meaning that they cannot be withdrawn to your wallet for any real economic value.
+
 Testing is possible through Ethereum Rinkeby  
 
-**Steps**:
+#### Steps
+
 - Deploy your vault with your own token
-- Mint your own tokens 
+- Mint your own tokens
 - Hack!
 
 ### What do I need for testing?
-When accessing each testnet, you will need to use its own native asset token. 
- 
-**Steps to test**: 
+
+When accessing each testnet, you will need to use its own native asset token.
+
+#### Steps to test
 
 - Go to https://hack.yearn.fi/
-- Switch to the testnet you wish to utilize over your wallet provider 
-- Make sure to have the native asset for the specific network 
+- Switch to the testnet you wish to utilize over your wallet provider
+- Make sure to have the native asset for the specific network
 - Get some tokens from the test client faucet (Ethereum Rinkeby https://faucet.rinkeby.io/)
```

Comment:
rinkeby is depreciated. hack.yearn.fi seems to default to goerli so this should probably link to a faucet for that? https://goerlifaucet.com/.

Reply:
yes please!